.travis.mk

```diff
@@ -82,7 +82,24 @@ docker_%:
 # commit, so the build requires old dependencies to be installed.
 # See ce623a23416eb192ce70116fd14992e84e7ccbbe ('Enable GitLab CI
 # testing') for more information.
-deps_debian:
+
+deps_tests:
+	pip install -r test-run/requirements.txt
+
+deps_ubuntu_ghactions: deps_tests
+	sudo apt-get update ${APT_EXTRA_FLAGS} && \
+		sudo apt-get install -y -f libreadline-dev libunwind-dev
+
+deps_coverage_ubuntu_ghactions: deps_ubuntu_ghactions
+	sudo apt-get install -y -f lcov
+	sudo gem install coveralls-lcov
+	# Link src/lib/uri/src to local src dircetory to avoid of issue:
+	# /var/lib/gems/2.7.0/gems/coveralls-lcov-1.7.0/lib/coveralls/lcov/converter.rb:64:in
+	#   `initialize': No such file or directory @ rb_sysopen -
+	#   /home/runner/work/tarantool/tarantool/src/lib/uri/src/lib/uri/uri.c (Errno::ENOENT)
+	ln -s ${PWD}/src src/lib/uri/src
```

Comment:
It surely should not be part of a `deps_*` goal. And it is strange to see this change in the commit that changes Debian to Ubuntu. Is it the regression from this commit or it was here before? Can we solve it separately?

I already saw something like this: 415c05868f95b45a56904d5aa581358aced05efe. But, to be honest, I don't know relative to what the `#line` directive should be.